web/packages/teleport/src/JoinTokens/JoinTokenForms.tsx

```diff
@@ -232,12 +254,30 @@ export const JoinTokenGCPForm = ({
   );
 };
 
+const supportedFieldsGcp = new Set([
+  '.allow.project_ids',
+  '.allow.locations',
+  '.allow.service_accounts',
+]);
+
+/**
+ * `checkGCPYAMLData` determines if the provided value contains item/s that are not supported by the edit form.
+ * @param data a value representing the gcp-specific config for a token
+ * @returns a boolean indicating if the provided value contains unsupported items
+ */
+export const checkGCPYAMLData = (data: unknown) => {
```

Comment:
The convention in our web code is that only the first letter of an acronym is capitalized. It should be Gcp, Yaml, Iam, etc.